vsintegration/src/FSharp.Editor/CompletionProvider.fs

```diff
@@ -43,7 +43,13 @@ type internal FSharpCompletionProvider(workspace: Workspace, serviceProvider: SV
     let xmlMemberIndexService = serviceProvider.GetService(typeof<IVsXMLMemberIndexService>) :?> IVsXMLMemberIndexService
     let documentationBuilder = XmlDocumentation.CreateDocumentationBuilder(xmlMemberIndexService, serviceProvider.DTE)
 
-    static member ShouldTriggerCompletionAux(sourceText: SourceText, caretPosition: int, trigger: CompletionTriggerKind, getInfo: (unit -> DocumentId * string * string list)) =
+    static member ShouldTriggerCompletionAux
```

Comment:
I'll do, but I don't see your point. Is this better?

![image](https://cloud.githubusercontent.com/assets/873919/20646099/da2ef85a-b484-11e6-828c-0c987d40d3dc.png)

I cannot even see how many parameters this method has and what's the result type.